2020/06.md

```diff
@@ -6,9 +6,10 @@
 - **Host**: Paypal
 - **Dates**: June 2 - 4, 2020
 - **Times**:
-  - 10:00 to 17:00 CT on June 2nd, 2020
-  - 10:00 to 17:00 CT on June 3rd, 2020
-  - 10:00 to 16:00 CT on June 4th, 2020
+  - 10:00 to 15:00 CT on June 1st, 2020
+  - 10:00 to 15:00 CT on June 2nd, 2020
+  - 10:00 to 15:00 CT CT on June 3rd, 2020
+  - 10:00 to 15:00 CT CT on June 4th, 2020
```

Comment:
```suggestion
  - 10:00 to 15:00 CT on June 3rd, 2020
  - 10:00 to 15:00 CT on June 4th, 2020
```

Reply:
Should we use America/Chicago to be more unambiguous? :)